src/components/passphrase/steps.js

```diff
@@ -0,0 +1,45 @@
+export default context => ({
+  info: {
+    cancelButton: {
+      title: 'cancel',
+      onClick: () => { context.props.closeDialog(); },
+    },
+    confirmButton: {
+      title: 'next',
+      onClick: () => { context.setState({ currentStep: 'generate' }); },
+    },
+  },
+  generate: {
+    cancelButton: {
+      title: 'cancel',
+      onClick: () => { context.props.closeDialog(); },
+    },
+    confirmButton: {
```

Comment:
The last step had also its own title - "Enter the missing word to continue". But there it isn't that big problem without it.